StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/PaymentSheetVerticalViewController.swift

```diff
@@ -150,6 +151,7 @@ class PaymentSheetVerticalViewController: UIViewController, FlowControllerViewCo
         loadResult: PaymentSheetLoader.LoadResult,
         isFlowController: Bool,
         analyticsHelper: PaymentSheetAnalyticsHelper,
+        walletButtonsShownExternally: Bool = false,
```

Comment:
Is there any reason to accept this in the init?  Seems like nobody is calling this with `true` ?

Reply:
We check it for `shouldShowApplePayInList` a few lines down. It's called by `FlowController.updateForWalletButtonsView()` with true when needed.

Reply:
Sorry, i was in the middle of writing this comment -- i meant to delete it. my bad!